### PR TITLE
Test/dashboard

### PR DESCRIPTION
**Pull request by stuartphilp**
reopening against dev, @pdehaan can you take a look?

### CODE REVIEW

File: dashboard/e2e-test/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Dashboard E2E tests
+
+End-to-end tests for the [Services QA Dashboard](https://github.com/mozilla-services/services-quality-dashboard)
+
+## How Do I Even?
+```sh
+$ git clone git@github.com:mozilla-services/services-test.git
+$ cd dashboard/e2etest
```
Comment (pdehaan) [inline dashboard/e2e-test/README.md:8]: @stuartphilp Should be `e2e-test`?

File: dashboard/e2e-test/config/stage.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  host: 'dashboard.stage.mozaws.net',
+  port: 80,
```
Comment (pdehaan) [inline dashboard/e2e-test/config/stage.js:3]: nit: trailing comma (although technically this isnt a pure JSON file so it's valid).

**Comment by pdehaan** [comment]
Apart from 2 nits, looks good!
:+1:

**Comment by stuartphilp** [comment]
thanks @pdehaan 

@silne30 care to take a look as well? it times out trying to connect to the browser. i'll probably just rewrite this in python

File: dashboard/e2e-test/tests/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/*eslint-env mocha */
```
Comment (cuff-links) [inline dashboard/e2e-test/tests/dashboard.js:1]: Do we 'use strict'; ?

Comment (cuff-links) [inline dashboard/e2e-test/tests/dashboard.js:1]: Apart from that, I think it all looks good.

**Pull request merged**